index.html

```diff
@@ -0,0 +1,53 @@
+<html>
+    <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>portfolio mobile</title>
+       <link rel="stylesheet" href="style.css">
+       <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    </head>
+ <body>   
+      <header class="navbar">
+        <div class="navbar__header">
+            <div class="navbar__header__content">
+              <div class="navbar__logo" href="#"> My Logo</div>
+                <div class="navbar__label_container">
+                  <span class="__label__list_container"></span>
+                  <span class="__label__list_container"></span>
+                  <span class="__label__list_container"></span></div>
+
+            </div>
```

Comment:
- [ ] As mentioned by the previous reviewer and for good semantics reasons, kindly ensure that your Hambarger is wrapped in a `nav` tag i.e. its container tag should be a `<nav>` tag.